uprotocol/transport/builder/uattributesbuilder.py

```diff
@@ -41,6 +41,7 @@ class UAttributesBuilder:
     """
 
     def __init__(self, id: UUID, type: UMessageType, priority: UPriority):
```

Comment:
Given that source is a required field, it should be added to the constructor.

Reply:
Added